tests/expectations/va_list.c

```diff
@@ -3,6 +3,25 @@
 #include <stdint.h>
 #include <stdlib.h>
 
-int32_t va_list_test(va_list ap);
+typedef int32_t (*VaListFnPtr)(int32_t count, ...);
 
-int32_t va_list_test2(va_list ap);
+typedef int32_t (*VaListFnPtr2)(int32_t count);
+
+typedef struct {
+  int32_t (*fn1)(int32_t count, ...);
+} Interface_______i32_______i32_______va_list;
+
+typedef struct {
+  int32_t (*fn1)(int32_t count);
+} Interface_______i32_______i32;
+
+int32_t va_list_test(int32_t count, ...);
+
+int32_t va_list_test2(int32_t count, ...);
+
+void va_list_fn_ptrs(int32_t (*fn1)(int32_t count, ...),
+                     int32_t (*fn2)(int32_t count),
```

Comment:
It looks like when used in function pointer the dotdotdot  version does not emit the va args dots in the C version. Is this intended?

Reply:
Created https://github.com/mozilla/cbindgen/pull/971

Reply:
I added this test to highlight the existing limitation.
I had attempted to address it with https://github.com/mozilla/cbindgen/pull/969 but there appears to have been some miscommunication.